lost/src/main/java/com/mapzen/android/lost/internal/GeofencingApiImpl.java

```diff
@@ -123,4 +129,16 @@ private void removeGeofences(LostApiClient client, String geofenceRequestId)
     return new SimplePendingResult(hasResult);
   }
 
+  public PendingIntent pendingIntentForIntentId(int intentId) {
+    return idToPendingIntent.get(intentId);
+  }
+
+  public Geofence geofenceForIntentId(int intentId) {
+    return idToGeofence.get(intentId);
+  }
+
+  public void setGeofenceForIntentId(int intentId, Geofence geofence) {
```

Comment:
It would be great if we could find a way to test this without adding a public method that is only called from tests. Is it possible to just have the tests call `addGeofence(...)`?